bftengine/src/bftengine/SigManager.cpp

```diff
@@ -225,19 +201,23 @@ uint16_t SigManager::getSigLength(PrincipalId pid) const {
   } else {
     std::shared_lock lock(mutex_);
     if (auto pos = verifiers_.find(pid); pos != verifiers_.end()) {
-      return pos->second->signatureLength();
+      auto result = pos->second->signatureLength();
+      return result;
     } else {
       LOG_ERROR(GL, "Unrecognized pid " << pid);
       return 0;
     }
   }
 }
 
-bool SigManager::verifySig(
+bool SigManager::verifyNonReplicaSig(
```

Comment:
verifyNonReplicaSig -> verifyClientSig ?

Reply:
Both names are inaccurate, this is the logic which uses the keys stored in signmanager (unlike keys which are stored in cryptomanager)

Reply:
Renamed